src/api/app/models/issue_tracker/cve_parser.rb

```diff
@@ -0,0 +1,51 @@
+# internal CVE parser class
+class IssueTracker::CVEParser < Nokogiri::XML::SAX::Document
+  @@my_tracker = nil
```

Comment:
I know you only moved code, but as you touched it... do you mind making this ruby? Class variables for states? Shudder

Reply:
@coolo I'm doing that right now, I just wrote the test cases before and now i will rewrite it :)